samply/src/linux_shared/mod.rs

```diff
@@ -399,7 +417,54 @@ where
         }
     }
 
+    /// This is a terrible hack to get binary correlation working with apps on Wine.
+    ///
+    /// Unlike ELF, PE has the notion of "file alignment" that is different from page alignment.
+    /// Hence, even if the virtual address is page aligned, its on-disk offset may not be. This
+    /// leads to obvious trouble with using mmap, since mmap requires the file offset to be page
+    /// aligned. Wine's workaround is straightforward: for misaligned sections, Wine will simply
+    /// copy the image from disk instead of mmaping them.
```

Comment:
Oh, but it also mmaps them first, otherwise this code wouldn't be called, right? I think this would be good to call out here. Reading the comment I was thinking that Wine would use `read` instead of `mmap` + `memcpy` for coping the image from disk, and then we wouldn't see it here. Unless I'm misunderstanding?

Reply:
Oh I see, it mmaps just the header. Yes, the comment addition you're proposing sounds perfect.

Reply:
I added the table and also clarified that PE header will be mapped first.